### PR TITLE
doc secrets for 2.8

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-8855

See PR for 2.9: https://github.com/stolostron/rhacm-docs/pull/5948/files

### CODE REVIEW

File: business_continuity/backup_restore/backup_arch.adoc [hunk 1]
```diff
@@ -64,6 +64,8 @@ kind: MyResource
     cluster.open-cluster-management.io/backup: cluster-activation
 ----
 
+*Note:* Regardless of the `cluster.open-cluster-management.io/backup` label value, secrets and config maps are restored when passive resources are restored. Since the `cluster-activation` value is ignored for these types of resources, set the `cluster.open-cluster-management.io/backup: cluster-activation` value to the resources using these secrets. This way, the resources are not restored until the activation step.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_arch.adoc]: I feel resources word is overused and makes this confusing.. I had to get back to jirra to clarify

What about 

*Note:* Regardless of the `cluster.open-cluster-management.io/backup` label value, secrets and config maps are always restored when passive resources are restored. Since the `cluster-activation` value is ignored in this case, set the `cluster.open-cluster-management.io/backup: cluster-activation` value to the resources using these secrets and config map **instead**, so that they are not restored until the activation step.

**Review by birsanv — CHANGES_REQUESTED** [review]

**Review by birsanv — APPROVED** [review]
lgtm ; possible change config map to config maps

**Review by birsanv — APPROVED** [review]
lgtm

File: business_continuity/backup_restore/backup_arch.adoc [hunk 2]
```diff
@@ -64,6 +64,8 @@ kind: MyResource
     cluster.open-cluster-management.io/backup: cluster-activation
 ----
 
+*Note:* Regardless of the `cluster.open-cluster-management.io/backup` label value, secrets and config maps are restored when passive resources are restored. Since the `cluster-activation` value is ignored in this case, set the `cluster.open-cluster-management.io/backup: cluster-activation` value to the resources using these secrets and config maps instead. This way, they are not restored until the activation step.
```
Comment (oafischer) [inline business_continuity/backup_restore/backup_arch.adoc]: There seem to be a lot of "this/these/they" in this paragraph, makes it a little hard to follow. If possible I'd try to be more specific.

I'd also move the relative clause after the main clause for better clarity. -> Secrets and config maps are restored when passive resources are restored, regardless of...

I also recommend using "because" instead of "since" to make translation easier.

**Review by oafischer — APPROVED** [review]
